docs/conan.md

```diff
@@ -0,0 +1,147 @@
+# Using dependencies from Conan
+
+[Conan](https://conan.io/) is a package manager for C/C++. We provide prebuilt binary dependencies for some platforms that are used by our CI, but they can also be consumed by users to build VCMI. However, it's not required to use only the prebuilt binaries: you can build them from source as well.
+
+## Supported platforms
+
+The following platforms are supported and known to work, others might require changes to our [conanfile.py](../conanfile.py) or upstream recipes.
+
+- **macOS**: x86_64 (Intel) - target 10.13 (High Sierra), arm64 (Apple Silicon) - target 11.0 (Big Sur)
+- **iOS**: arm64 - target 12.0
+
+## Getting started
+
+1. [Install Conan](https://docs.conan.io/en/latest/installation.html)
+2. Execute in terminal: `conan profile new default --detect`
+
+## Download dependencies
+
+0. If your platform is not on the list of supported ones or you don't want to use our prebuilt binaries, you can still build dependencies from source or try consuming prebuilt binaries from the central Conan repository - [ConanCenter](https://conan.io/center/). In this case skip to the next section directly.
+
+1. Check if your build environment can use the prebuilt binaries: basically, that your compiler version (or Xcode major version) matches the information below. If you're unsure, just skip to next step.
```

Comment:
"If you're unsure, just skip to next step"
Which step should I skip? In that step there is no actions as I get it

Reply:
Next after 1 is 2 :)

Basically it means that you just try to use the pre-built deps and see if Conan complains or not.

Reply:
it confuses, 1 doesn't suppose any action from me, not clear what should I skip

Reply:
there's a verb "check" - it means action

Reply:
Whatever I check/sure or not, I still have to go next step

Reply:
if you checked and are sure you can't use our prebuilt binaries, it obviously makes no sense to continue with these steps - that's the whole point of the check. If you're unsure, you just continue with these steps and then you'll see if you can use the binaries or not (whether `conan install` fails or not).